package.json

```diff
@@ -32,6 +32,7 @@
     "react-dom": "^18",
     "react-hook-form": "^7.52.0",
     "react-icons": "^5.2.1",
+    "sass": "^1.77.6",
```

Comment:
installed SASS for more custom styling, now if you want to write custom styles go to `/app/styles/`, create a `_[page-name].scss` and import it into `main.scss`